src/api/app/helpers/webui/user_helper.rb

```diff
@@ -1,4 +1,36 @@
 module Webui::UserHelper
+  def display_applied_filters(filters)
+    tag.span(class: 'd-block ml-4 mb-3') do
+      link_to(User.session!) do
+        tag.i(class: 'fas fa-trash-alt') do
+          tag.b do
+            "Clear current search #{display_filters(filters.keys).to_sentence}"
+          end
+        end
+      end
+    end
+  end
+
+  def display_filters?(filters)
+    return false if filters.empty?
+    return false if filters.size == 1 && filters.include?('search_text')
+
+    true
+  end
```

Comment:
I think we can get rid of this method and use a simplified condition in the view. See comment there.

Reply:
hm, i prefer to keep the method but add the suggested logic to it.